swift/Sources/FlatBuffers/FlatBufferBuilder.swift

```diff
@@ -469,6 +451,7 @@ public struct FlatBufferBuilder {
     ///   - condition: Condition to insert
     ///   - def: Default condition
     ///   - position: The predefined position of the element
+    @available(*, deprecated, message: "Please Re-Generate the code")
```

Comment:
Add a clearer message example: `Deprecated, function will be removed in Flatbuffers 0.6.0`